.tekton/assets/sidecars.json

```diff
@@ -22,16 +22,24 @@
     },
     {
       "name": "mongodb",
-      "image": "mongo:8.0.8"
+      "image": "public.ecr.aws/docker/library/mongo:8.0.8"
     },
     {
       "name": "mssql",
-      "image": "mcr.microsoft.com/mssql/server:2022-RC1-ubuntu-20.04",
```

Comment:
We had to update to Ubuntu 24. The test was no longer working with this ^ version. Updated to 2022